src/components/dataset/schemaCreation/DatasetSchemaInformationView.tsx

```diff
@@ -0,0 +1,334 @@
+            pattern: {
+              value: /^[a-zA-Z0-9][_a-zA-Z0-9]*$/,
+              message: 'Name should fit pattern: ^[a-zA-Z0-9][_a-zA-Z0-9]*$',
+            },
+          })}
+        />
+      </Grid>
+
+      <Grid item xs={12} className={classes.formFieldContainer}>
+        <label
+          htmlFor="dataset-description"
+          className={clsx(classes.formLabel, { [classes.formLabelError]: errors.description })}
+        >
+          Dataset description*
+        </label>
+        <Controller
+          name="description"
+          control={control}
+          rules={{ required: 'description is required' }}
+          render={({ field }) => (
+            <>
+              <SimpleMdeReact options={editorOptions} {...field} />
+              {errors.description && (
+                <span className={classes.formInputError}>{errors.description.message}</span>
+              )}
+            </>
+          )}
+        />
+      </Grid>
+
+      <Grid item xs={6}>
+        <label
+          htmlFor="dataset-terraProject"
+          className={clsx(classes.formLabel, { [classes.formLabelError]: errors.terraProject })}
+        >
+          Terra project*
+        </label>
+        <Controller
+          name="terraProject"
+          control={control}
+          render={({ field }) => (
+            <Select id="dataset-terraProject" className={classes.formInput} {...field}>
+              <MenuItem value="yes">Yes</MenuItem>
+              <MenuItem value="no">No</MenuItem>
+            </Select>
+          )}
+        />
+      </Grid>
+
+      <Grid item xs={6}>
+        <label htmlFor="dataset-enableSecureMonitoring" className={classes.formLabel}>
+          Secure monitoring
+        </label>
+        <Controller
+          name="enableSecureMonitoring"
+          control={control}
+          render={({ field }) => (
+            <Select id="dataset-enableSecureMonitoring" className={classes.formInput} {...field}>
+              <MenuItem value="true">Yes</MenuItem>
+              <MenuItem value="false">No</MenuItem>
+            </Select>
+          )}
+        />
+      </Grid>
+
+      <Grid item xs={6}>
+        <label htmlFor="dataset-cloudPlatform" className={classes.formLabel}>
+          Cloud Platform*
+        </label>
+        <Controller
+          name="cloudPlatform"
+          control={control}
+          rules={{ required: true }}
+          render={({ field }) => (
+            <Select
+              id="dataset-cloudPlatform"
+              className={classes.formInput}
+              {...field}
+              onChange={(event: any, change: any) => {
+                const cloudPlatform = event.target.value;
+                setRegionOptions(cloudPlatform === 'gcp' ? GCP_REGIONS : AZURE_REGIONS);
+                field.onChange(event, change);
+              }}
+            >
+              <MenuItem value="gcp">Google Cloud Platform (GCP)</MenuItem>
```

Comment:
Could "gcp" be abstracted into a constant?

Reply:
Updated! There was already a constant for this! :D